tests/contract/suggestion_contract_test.py

```diff
@@ -57,6 +53,38 @@ def test_suggestion_concepts(self, api, wait_for_ready):
             else:
                 assert has_partial_match_in_title(hit['prefLabel'], prefix), "Title without match on prefix encountered "
                 was_partial_count += 1
+
+    @pytest.mark.contract
+    def test_suggestion_dataservices(self, api, wait_for_ready):
+        prefix = "Swag"
+        result = requests.get(suggestions_endpoint + '/dataservices?q=Swag'.format(prefix))
+        assert result.status_code == 200
+        previous_was_prefix = True
+        was_prefix_count = 0
+        was_partial_count = 0
+        for hit in result.json()['suggestions']:
```

Comment:
legg til test på  length for suggestions, ellers vil det gå igjennom om det ikke er noen forslag